types/recordrtc/recordrtc-tests.ts

```diff
@@ -9,6 +9,15 @@ navigator.getUserMedia(
             type: 'video',
             disableLogs: true,
             bufferSize: 2048,
+            ondataavailable: (blob: Blob) =>
```

Comment:
The CI tests are failing because you're missing a `{` at the end of this line

Reply:
Oops, I did that on my phone yesterday where the GitHub UI has neither syntax highlighting nor auto completion. Sorry for that!